src/main/kotlin/org/rust/cargo/runconfig/customBuild/CustomBuildConfigurationType.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.runconfig.customBuild
+
+import com.intellij.execution.configurations.ConfigurationFactory
+import com.intellij.execution.configurations.ConfigurationTypeBase
+import com.intellij.execution.configurations.ConfigurationTypeUtil
+import com.intellij.execution.configurations.RunConfiguration
+import com.intellij.openapi.project.Project
+import org.rust.cargo.icons.CargoIcons
+
+class CustomBuildConfigurationType : ConfigurationTypeBase(
+    "CustomBuildRunConfiguration",
+    "custom-build",
```

Comment:
Is `custom-build` name really recognizable among Rust programmers?  Also, it doesn't seems to be a proper style for a run configuration category name

![image](https://user-images.githubusercontent.com/3221931/226582408-44d3b0df-b54d-4af2-b47b-8af8a148d771.png)